charts/clickhouse/Chart.yaml

```diff
@@ -12,6 +12,6 @@ keywords:
 name: clickhouse
 sources:
   - https://github.com/sentry-kubernetes/charts
-version: 3.9.0
+version: 3.10.0
```

Comment:
Could you remove that change please?

Reply:
Done! Thought that was needed to satisfy the CI.